cocos/core/event/eventify.ts

```diff
@@ -158,7 +155,7 @@ export function Eventify<TBase> (base: Constructor<TBase>): Constructor<TBase &
             if (!callback) {
                 this.removeAll(type);
             } else {
-                this._callbacksInvoker.off(type, callback, target);
+                CallbacksInvoker.prototype.off.call(this, type, callback, target);
```

Comment:
Can we cache CallbacksInvoker.prototype.xxx rather than access every time?

Reply:
Definitely yes: https://github.com/cocos-creator/engine/pull/6782/commits/d0c59f831310a75ccab575a3b96f6967a2381004